v0.5/mlperf.conf

```diff
@@ -3,6 +3,18 @@
 # Model maybe '*' as wildcard. In that case the value applies to all models.
 # All times are in milli seconds
 
+# set performance_sample_count for each model
+mobilenet.*.performance_sample_count_override = 1024
```

Comment:
Is this the correct way to set perf_count? I thought this option is for audit purpose. @georgelyuan 

The potential issue is, QSL does not know that LoadGen has secretly override the perf_count.

Reply:
Would that be a problem for the QSL? This is technically a performance setting, not a QSL/dataset setting
@DilipSequeira

Reply:
This should work for all models except for gnmt, as @nvpohanh points out below.

For gnmt, we want the performance sample count to be conditional on whether we are in accuracy or performance mode. Unconditionally making performance count 3903900 will make it larger than the total sample count of 3003 in accuracy mode. The loadgen probably does not handle that properly. (Inspecting now.)

Reply:
but aren't we supposed to use `.large` file for GNMT performance mode? In that case, the sample count would be 3903900. Am I correct?

Reply:
.large is not defined:
https://github.com/mlperf/policies/blob/master/submission_rules.adoc#552-inference

Reply:
Sorry, I was referring to the `newstest2014.tok.bpe.32000.en.large`. The reference implementation uses `newstest2014.tok.bpe.32000.en.large` for perf run and `newstest2014.tok.bpe.32000.en.large` for accuracy run.
https://github.com/mlperf/inference/blob/master/v0.5/translation/gnmt/tensorflow/loadgen_gnmt.py#L476

Reply:
got it. I guess the gnmt reference does not user mlperf.conf and instead does this:
```
perf_queries = min(total_queries, 3003) 
```
which would shield loadgen. Guess we need to check what loadgen does if ```total < perf_queries```.

Btw, 3003 should be 3903900 ?

Reply:
This is what #481 was trying to fix.

Reply:
I wish we'd take #481.